test/docs.js

```diff
@@ -443,6 +377,103 @@ describe('Docs', function() {
     assert(larryDoc.relatedReverseIds.length === 1);
     assert(larryDoc.relatedReverseIds[0] === 'paul');
     assert(larryDoc.cacheInvalidatedAt.getTime() === response.updatedAt.getTime());
+
+    apos.doc.db.deleteMany({ slug: { $in: [ 'paul', 'carl', 'larry' ] } });
+  });
+
+  it('should remove the related reverse IDs when you delete a draft document', async function () {
```

Comment:
Testing that deleting a draft properly remove related reverse IDs to draft and published docs.